src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -18,7 +18,7 @@ interface AttentionBoxProps extends VibeComponentProps {
   // Will remove when releasing version 2 as BREAKING CHANGES
   withIconWithoutHeader?: boolean;
   /** we support 4 types of attention boxes */
-  type?: AttentionBoxTypes;
+  type?: AttentionBoxType;
   /** Icon classname for icon font or SVG Icon Component for SVG Type */
   icon?: string | React.FunctionComponent | null;
```

Comment:
I'm making icons everywhere with this type `string | React.FC<iconSubComponentProps> | null`, which I've seen somewhere. Wouldn't it be better to use it here also?

Reply:
You are right, I will change it